python/neuroglancer/pipeline/task_creation.py

```diff
@@ -51,7 +51,7 @@ def compute_bigarray_bounding_box(storage):
     abs_x_min = abs_y_min = abs_z_min = float('inf')
     abs_x_max = abs_y_max = abs_z_max = 0
     for filename in tqdm(storage.list_files(prefix='bigarray/')):
-        match = re.match(r'(\d+):(\d+)_(\d+):(\d+)_(\d+):(\d+)$', filename)
+        match = re.search(r'(\d+):(\d+)_(\d+):(\d+)_(\d+):(\d+)$', filename)
```

Comment:
`match` looks from the beginning, `search` looks in the entire string. The big change is that now instead of returning `0_64-0_64-0_64`, when a prefix is specified, it returns: `bigarray/0_64-0_64-0_64`. Without this change, the code breaks.